azurerm/resource_arm_application_gateway.go

```diff
@@ -49,6 +49,33 @@ func resourceArmApplicationGateway() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"identity": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"type": {
+							Type:     schema.TypeString,
+							Required: true,
```

Comment:
Would it make sense to mark this as `Optional` with a `Default`

Reply:
Hi @katbyte , yep that makes sens, done. Also fixed the acc tests.